File: 01-algaritmos-basicos/03-memoria_estrutura_repeticao/exercicios/em_java/Primo.java
import java.util.Locale;
import java.util.Scanner;

public class Primo {

    public static void main(String[] args) {
	
	   Scanner sc = new Scanner(System.in);
	   Locale.setDefault(Locale.US);
	   
	   int x;
	   int primo = 0;
	   int naoPrimo = 0;
	   
	   System.out.print("Digite um numero: ");
	   x = sc.nextInt();
	   
	   for(int i = 1; i <= x; i++) {
	   
			if( x % 2 == 0 ) {
				primo++;
			} else {
				naoPrimo++;
			}
	
	   }
		if(primo > 2 || x == 1) {
			System.out.printf("%d Nao e primo%n", x);
		} else {
			System.out.printf("%d E primo%n", x);
		}
		
	}

}

File: 01-algaritmos-basicos/03-memoria_estrutura_repeticao/exercicios/em_C/primo.c
#include <stdio.h>

int main() {
	
	printf("Exercio 4\n");
	
	int numero;
	int primo = 0;
	int naoPrimo = 0;
	printf("Digite um numero inteiro: ");
	scanf("%d", &numero);
	
	for(int i = 1; i <= numero; i++) {

		if(numero % i == 0) {
			
			primo++;
			
		} else {

			naoPrimo++;
		
		}
	
	}
	
	if(primo > 2 || numero == 1) {
			
		printf("%d nao e primo", numero);
		
	} else {

		printf("%d e primo", numero);
	
	}
	
	
	return 0;
}